ESMF/Aerosol_GridComp/AeroCinderella_GridComp/Cinderella/CMakeLists.txt

```diff
@@ -6,14 +6,13 @@ set (srcs
         ConvertUnits.F90
         NOAA_TracerMap.F90
         NOAA_GOCARTtracers.F90
-        MissingFields.F90
+#       MissingFields.F90
```

Comment:
If MissingFields.F90 is no longer needed, should it be deleted?

Reply:
The Cinderella component is not used at all in this implementation. I generally agree with deleting unnecessary code, however I will defer to you for the decision. Note that this PR includes code under active development, so changes to the code structure are still possible.

Reply:
@rmontuoro Since the Cinderella component is no longer being used in this implementation, and it is not being used by GEOS then it can be eliminated also. @tclune do you agree?

Reply:
Commit 195f02c removed the Cinderella component.